### PR TITLE
Added postgres 13 compatibility to GA release notes

### PR DESCRIPTION
**Pull request by ericharmeling**
Fixes https://github.com/cockroachdb/docs/issues/10607.

### CODE REVIEW

File: v20.2/postgresql-compatibility.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ toc: true
 redirect_from: porting-postgres.html
 ---
 
-CockroachDB supports the PostgreSQL wire protocol and the majority of its syntax. This means that your existing applications can often be migrated to CockroachDB without changing application code.
+CockroachDB supports v3.0 of the [PostgreSQL wire protocol](https://www.postgresql.org/docs/current/protocol.html) and the majority of PostgreSQL syntax. This means that existing applications built on PostgreSQL can often be migrated to CockroachDB without changing application code.
```
Comment (vy-ton) [inline v20.2/postgresql-compatibility.md:8]: `v13`

File: v21.1/postgresql-compatibility.md [hunk 2]
```diff
@@ -5,7 +5,7 @@ toc: true
 redirect_from: porting-postgres.html
 ---
 
-CockroachDB supports the [PostgreSQL wire protocol](https://www.postgresql.org/docs/current/protocol.html) and the majority of its syntax. This means that your existing applications can often be migrated to CockroachDB without changing application code.
+CockroachDB supports v3.0 of the [PostgreSQL wire protocol](https://www.postgresql.org/docs/current/protocol.html) and the majority of PostgreSQL syntax. This means that existing applications built on PostgreSQL can often be migrated to CockroachDB without changing application code.
```
Comment (vy-ton) [inline v21.1/postgresql-compatibility.md:8]: `v13`

**Review by vy-ton — APPROVED** [review]

File: v20.2/postgresql-compatibility.md (hunk 1 above)
Comment (ericharmeling) [reply v20.2/postgresql-compatibility.md:8]: The version of the protocol that we support is v3.

From https://www.postgresql.org/docs/current/protocol.html:

> https://www.postgresql.org/docs/current/protocol.html

Comment (vy-ton) [reply v20.2/postgresql-compatibility.md:8]: Sounds good, didn't know they didnt keep the same versioning

Comment (ericharmeling) [reply v20.2/postgresql-compatibility.md:8]: > didn't know they didnt keep the same versioning 

Yeah IDK why they don't - just makes it more confusing. TFTR!

**Pull request merged**